pages/en/lb3/Polymorphic-relations.md

```diff
@@ -45,7 +45,38 @@ The examples below use three example models: Picture, Author, and Reader, where
 
 ## HasMany polymorphic relations
 
-The usual options apply, for example: `as: 'photos'` to specify a different relation name/accessor.
+Take the following scenario as example:
+
+> A Picture model belongs to either an Author model or a Reader model 
+
+A hasMany polymorphic relation means: 
+
+- There are two properties created in model Picture, one serves as a discriminator, which states what model the picture belongs to, and the other one servers as a foreignKey, which is the id(or primaryKey) value of either the author or reader. 
+
+- There are CRUD apis added to modelFrom(Author and Reader), by which you can create/modify/delete modelTo instance that belongsTo it. For example, `author.pictures.create()` create a new picture belongsTo author.
```

Comment:
@b-admike lower case `picture` and `author` are already instance :p  Camel case `Picture` and `Author` means model, and I think here a model instance belongs to another model instance(both of them are instantiated), not a model.